spec/dummy/app/views/alchemy/page_layouts/_search.html.erb

```diff
@@ -1,2 +1,3 @@
+<% @search_results = Alchemy::Search::SearchPage.perform_search(params, ability: current_ability) %>
 <%= render_search_form %>
-<%= render_elements %>
+<%= render_search_results %>
```

Comment:
```suggestion
<%= render "alchemy/search/results", search_results: search_results %>
```

Reply:
And adjust the `alchemy/search/results` partial so it loads the results from the local vars instead of the ivar

Reply:
and remove the `render_search_results` helper